src/services/CacheService/index.ts

```diff
@@ -0,0 +1,78 @@
+import { ONE_DAY_TIMESTAMP } from '../../constants/common'
+import CONFIG from '../../config'
+import { PersistenceService, persistenceService } from '../PersistenceService'
+
+export const NAMESPACE_CACHE = 'cache'
+
+export interface CacheOptions {
+  expire?: number
+  // When expire is present, ignore expire.
+  expireAt?: number
+  allNetworkSharing?: boolean
```

Comment:
This key is not intuitive. How about `networkTypeAgnostic`

Reply:
Given an understanding of the business logic, `networkTypeAgnostic` would be more appropriate, but in the absence of knowledge of the business logic, `allNetworkSharing` might be easier to guess?

I think I need to add some comments to it.

Reply:
done